src/framework/components/camera/component.js

```diff
@@ -289,6 +289,10 @@ class CameraComponent extends Component {
         }
     }
 
+    set renderSceneColorMap(value) {
+        this.requestSceneColorMap(value);
```

Comment:
I'm not sure it's correct this just calls requestSceneColorMap with the value.

In most cases (default), this will be false, and it will cause the internal counter to be decremented to -1.

So if something else would set it to true, the counter would be incremented to 0, and it would still not render, as it needs to be > 0.

Reply:
True, we should keep track of the request cast by the component internally, doing that.